src/libs/ajax/leonardo/Runtimes.ts

```diff
@@ -186,13 +224,14 @@ export const Runtimes = (signal: AbortSignal) => {
       // [IA-3710] In order to keep the front-end backwards compatible, any Azure tool labels
       // will be changed to JupyterLab.
       const runtimeList = await res.json();
-      const runtimesWithToolLabelDecorated = _.map((runtime) => {
+      const runtimesWithToolLabelDecorated: RawListRuntimeItem[] = _.map((runtime) => {
```

Comment:
This could, I think, become something we move into `getNormalizedListRuntime`. But not in this PR